qrenderdoc/Code/QRDUtils.cpp

```diff
@@ -480,6 +480,36 @@ void RichResourceTextInitialise(QVariant &var, ICaptureContext *ctx)
 
     linkedText->ctxptr = ctx;
 
+    // convert URL into clickable links
+    auto handleURLFragment = [&linkedText](QString str) {
```

Comment:
Please remove the lambda capture here and add a parameter that is the `RichResourceTextPtr`. That way this will definitely just be compiled to a simple function and not a variable, even in debug builds (possibly in release the compiler will optimise that out as it's local, but debug performance is important too).